setup.py

```diff
@@ -6,6 +6,8 @@
 
 from graxpert.version import release, version
 
+sys.setrecursionlimit(15_000)

```

Comment:
In windows cx_freeze crashes without setting an higher recursion limit using onnxruntime